actors/multisig/src/state.rs

```diff
@@ -35,8 +36,8 @@ pub struct State {
 
 impl State {
     /// Checks if `address` is in the list of signers
-    pub fn is_signer(&self, address: &Address) -> bool {
-        self.signers.contains(address)
+    pub fn is_signer(&self, id: &ActorID) -> bool {
+        self.signers.contains(&Address::new_id(*id))
```

Comment:
I think this is to act as a level of protection to ensure that we don't do bogus checks of non id addresses against addresses that are resolved to id addresses.

If we go this route why not also do this on purge_approvals?  This also implies we should add st accessors for push, retain etc that take ActorIDs and do the address wrapping internal to the state.

You could also argue that this is unnecessary and then this could go back to taking in an address.  It would be good to be consistent.